project/Dependencies.scala

```diff
@@ -1,7 +1,7 @@
 import sbt._
 
 object Dependencies {
-  val akkaV = "2.3.6"
+  val akkaV = "2.5.3"
```

Comment:
TestKit is now failing with `java.lang.NoSuchMethodError: akka.actor.ActorSystem.shutdown()` when running tests.  I'm guessing it's due to this.